src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -24,7 +24,8 @@
     <UsePublicApiAnalyzer>false</UsePublicApiAnalyzer>
     <GetSigningInputsDependsOn>GetSigningInputsCustom</GetSigningInputsDependsOn>
     <GetSymbolsToIndexDependsOn>GetSymbolsToIndexCustom</GetSymbolsToIndexDependsOn>
-</PropertyGroup>
+    <NoWarn>$(NoWarn);CS1591;IDE0059;IDE1006;IDE0049</NoWarn>
```

Comment:
CS1591 is related to XMLDoc, and therefore the title of this PR, but the other 3 codes are not.

We want to keep IDE1006, so disabling for the whole project feels wrong. I know we've got a bunch of noise on static properties, but we want to keep warnings (ideally enforce) public properties being Pascal Case, private members being _camelCase, methods being Pascal Case.

An in my opinion, IDE0059 seems like a good analyzer we should keep.

I think IDE0049 is very reasonable too, though we've got a lot of violations in our repo. But suppressing the code increases risk that future code changes will continue to add more violations, making it harder to enforce in the future.